ingredient/ingredient-container-reg/src/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Can you move this up to the root folder, and update src to point to src folder. We refactored all the projects for tsconfig to live at project root. You can see that in any ingredient under latest master.

Reply:
Moved tsconfig and fixed references.